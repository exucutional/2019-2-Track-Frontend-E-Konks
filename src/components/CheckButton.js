/* eslint-disable react/destructuring-assignment */
import React from 'react';
import PropType from 'prop-types';
import styled from '@emotion/styled';

const Button = styled.img`
	display: flex;
	height: 8vh;
	margin-right: 10px;
	filter: invert(1);
	&:hover {
		filter: invert(0.5);
	};
	cursor: pointer;
`;

function CheckButton(props) {
	return <Button onClick={ props.onClick } src="https://image.flaticon.com/icons/svg/446/446191.svg"/>;
}

CheckButton.propTypes = {
	onClick: PropType.func.isRequired,
}

export default CheckButton;

