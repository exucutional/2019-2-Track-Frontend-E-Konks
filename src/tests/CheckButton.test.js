import React from 'react';
import renderer from 'react-test-renderer';
import CheckButton from '../components/CheckButton'

it('renders correctly', () => {
	const tree = renderer
		.create(<CheckButton onClick={ () => {} }/>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
