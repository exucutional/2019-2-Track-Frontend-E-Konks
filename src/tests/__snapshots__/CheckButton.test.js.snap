// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-mq31ep"
  id="check-button"
  onClick={[Function]}
  src="https://image.flaticon.com/icons/svg/446/446191.svg"
/>
`;
